### PR TITLE
update ha page and images

### PR DESCRIPTION
**Pull request by evilnick**
woke-checker isn't so good at images

### CODE REVIEW

File: pages/k8s/high-availability.md [hunk 1]
```diff
@@ -32,18 +32,18 @@ a better option.
...
+control plane nodes with  `juju add-unit kubernetes-control-plane`:
 
-![multi-master worker image][img-multi-master]
+![multi-control plane worker image][img-multi-control-plane]
 
 While this will add more machines, it doesn't work as may be expected. What
-happens is that the workers will randomly pick a master to communicate with and
-always use that master unit. This means if that master fails in a way that
+happens is that the workers will randomly pick a control plane to communicate with and
+always use that control plane unit. This means if that control plane fails in a way that
```
Comment (Cynerva) [inline pages/k8s/high-availability.md:41]:
```suggestion
happens is that the workers will randomly pick a control plane unit to communicate with and
always use that unit. This means if that control plane unit fails in a way that
```

File: pages/k8s/high-availability.md [hunk 2]
```diff
@@ -32,18 +32,18 @@ a better option.
...
 doesn't remove it from **Juju**, those workers are simply unable to communicate
-with the control plane. If workers arbitrarily pick the same master, they can
-also overload the master with traffic, making the additional units redundant.
+with the control plane. If workers arbitrarily pick the same control plane, they can
+also overload the control plane with traffic, making the additional units redundant.
```
Comment (Cynerva) [inline pages/k8s/high-availability.md:44]:
```suggestion
with the control plane. If workers arbitrarily pick the same control plane unit, they can
also overload the unit with traffic, making the additional units redundant.
```

File: pages/k8s/high-availability.md [hunk 3]
```diff
@@ -32,18 +32,18 @@ a better option.
...
 
-Load balancing the masters is the next logical step:
+Load balancing the control planes is the next logical step:
```
Comment (Cynerva) [inline pages/k8s/high-availability.md:46]:
```suggestion
Load balancing the control plane is the next logical step:
```

File: pages/k8s/high-availability.md [hunk 4]
```diff
@@ -54,11 +54,11 @@ relevant "Cloud Integration" section of the documentation for more details
...
 The workers now all use the load balancer to talk to the control plane. This
-will balance the load to the master units, but we have just moved the single
+will balance the load to the control plane units, but we have just moved the single
 point of failure to the load balancer. Floating a virtual IP address in front
-of the master units works in a similar manner but without any load balancing.
-If your cluster won't generate enough traffic to saturate a single master, but
-you want high availability on the control plane, multiple masters floating a
+of the control plane units works in a similar manner but without any load balancing.
+If your cluster won't generate enough traffic to saturate a single control plane, but
+you want high availability on the control plane, multiple control planes floating a
```
Comment (Cynerva) [inline pages/k8s/high-availability.md:61]:
```suggestion
you want high availability on the control plane, multiple control plane units floating a
```

**Review by Cynerva — CHANGES_REQUESTED** [review]
This doc often shortened "master unit" to simply "master". With these updates, it shortens "control plane unit" to "control plane" in the same way. I think that could lead to confusion since "control plane" usually refers to the whole control plane including all kubernetes-control-plane units.

See my suggested changes that I hope will clear this up a bit.

**Comment by evilnick** [comment]
@Cynerva thanks for that, I guess I should have given it another readthrough but all those masters and control planes just started sounding like gobbledygook at some point

**Review by Cynerva — APPROVED** [review]

**Pull request merged**